### PR TITLE
add org.gnome.ColorProfileViewer

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by nedrichards** [comment]
This won't build on flathub, your file name has to be the same as your app ID. I don't think it's *supposed* to be org.gnome.PowerStats ;-)

**Comment by hughsie** [comment]
@nedrichards gahh, missed that sorry. It's actually supposed to be ColorProfileViewer -- two PRs in one day has confused me and you both :)

File: org.gnome.ColorProfileViewer.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
+{
+  "app-id": "org.gnome.ColorProfileViewer",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gcm-viewer",
+  "branch": "stable",
```
Comment (nedrichards) [inline org.gnome.ColorProfileViewer.json]: this bit is redundant on flathub and fwiw I find it annoying to have specified for local dev (where I generally want my version to be 'master'

**Comment by nedrichards** [comment]
bot, build org.gnome.ColorProfileViewer
testbot, build org.gnome.ColorProfileViewer

**Comment by hughsie** [comment]
Oops, patch was missing.  I'll try commanding the bot:

bot, build org.gnome.ColorProfileViewer
testbot, build org.gnome.ColorProfileViewer

**Comment by nedrichards** [comment]
bot, build org.gnome.ColorProfileViewer

**Comment by nedrichards** [comment]
i needed more dbus holes to make it work. Specifically this command improved things:
> flatpak run --log-session-bus --talk-name=org.gtk.vfs --talk-name=org.gtk.vfs.* --own-name=org.gnome.ColorManager.Viewer org.gnome.ColorProfileViewer

should the app id actually be `org.gnome.ColorManager.Viewer`? Or is that legacy in some way.

**Comment by hughsie** [comment]
> should the app id actually be `org.gnome.ColorManager.Viewer`

That's a good question. gnome-color-manager the project is currently a mixture of the session calibration tools and the color profile viewer, mixed in with various tools used by gnome-control-center. My current feeling is I should split out the "viewer" app into a completely different repo.

**Comment by hughsie** [comment]
Okay, lets close this and work on a fresh PR now I've split out the viewer from the rest of the muddle in gnome-color-manager.

**Comment by hughsie** [comment]
Okay, continuing in https://github.com/flathub/flathub/pull/854 -- thanks for the help so far.

**Pull request closed without merging**